eclair-node-gui/src/main/scala/fr/acinq/eclair/gui/GUIUpdater.scala

```diff
@@ -86,7 +86,7 @@ class GUIUpdater(mainController: MainController) extends Actor with ActorLogging
       val (channelPaneController, root) = createChannelPanel(channel, peer, remoteNodeId, isFunder, channelId)
       channelPaneController.updateBalance(currentData.commitments)
       val m1 = m + (channel -> channelPaneController)
-      val totalBalance = MilliSatoshi(m1.values.map(_.getBalance.amount).sum)
+      val totalBalance = MilliSatoshi(m1.values.map(_.getBalance.toLong).sum)
```

Comment:
This can use `sum` on the `MilliSatoshi` class since we also have defined `NumericMilliSatoshi extends Numeric[MilliSatoshi]`

Reply:
Good catch!